modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/RosettaSearch.scala

```diff
@@ -0,0 +1,20 @@
+package org.tessellation.rosetta.server.model
+
+import org.tessellation.schema.transaction.Transaction
+
+object RosettaSearch {
+
+  def searchBySourceAddress(transactionHash: String, snapshots: List[Transaction]) =
```

Comment:
Why is the `snapshots` parameter of type `List[Transaction]`?

Reply:
This object is unneeded. Removed.